yt_dlp/extractor/crunchyroll.py

```diff
@@ -146,24 +146,26 @@ def _extract_chapters(self, internal_id):
         # if no skip events are available, a 403 xml error is returned
         skip_events = self._download_json(
             f'https://static.crunchyroll.com/skip-events/production/{internal_id}.json',
-            internal_id, note='Downloading chapter info', fatal=False, errnote=False) or {}
+            internal_id, note='Downloading chapter info', fatal=False, errnote=False)
+        if not skip_events:
+            return None
 
         chapters = []
         for event in ('recap', 'intro', 'credits', 'preview'):
-            start = skip_events.get(event, {}).get('start')
-            end = skip_events.get(event, {}).get('end')
-
+            start = traverse_obj(skip_events, (event, 'start', {float_or_none}))
+            end = traverse_obj(skip_events, (event, 'start', {float_or_none}))
```

Comment:
Should be ` end = traverse_obj(skip_events, (event, 'end', {float_or_none}))` - 'start' is wrong here.